usecases/cs/lpc/usecase.go

```diff
@@ -204,6 +208,8 @@ func (e *LPC) AddFeatures() {
 			},
 		}
 		_ = lc.UpdateLimitDataForIds(newLimiData)
+	} else {
```

Comment:
With these long if statements, it might be easier to read the code if the error and return are handled directly in the if statement.

So changing instances like this into 

```golang
dcs, err := ...
if err != nil { 
  return err
}
```

That also removes the need for an if statement for success.

In addition I think we should check very early for feature existence and exit before adding any data if any of the required features are missing. This applies to all modified usecase.go implementations.